src/Compilers/CSharp/Portable/Symbols/AssemblySymbol.cs

```diff
@@ -466,6 +468,18 @@ internal bool RuntimeSupportsNumericIntPtr
             }
         }
 
+        /// <summary>
+        /// Figure out if the target runtime supports inline array types.
```

Comment:
>Figure out if the [](http://example.com/codeflow?start=12&length=17)

Consider updating the comments on each of the `Runtime...` properties to "Returns true if ..." or "True if ...".